### ISSUE TITLE
Variables from `\GetIdInfo` are not declared

### ISSUE THREAD

**Issue by u-fischer** [latex3/latex3#620]
When using check-declarations with a package which uses `\GetIdInfo` it gives errors for all the \ExplFile-variables:

```
\begin{filecontents}{testexplsvn.sty}
\RequirePackage{expl3}
\GetIdInfo$Id: testexplsvn.dtx 0.1 2019-09-01 12:00:00Z UF $ {some desc}
\ProvidesExplPackage
  {\ExplFileName}{\ExplFileDate}{\ExplFileVersion}{\ExplFileDescription}
\end{filecontents}  
\PassOptionsToPackage{enable-debug,check-declarations}{expl3}
\documentclass{article}
\usepackage{testexplsvn}
\begin{document}
abc
\end{document}
```

```
! LaTeX3 Error: The variable \ExplFileDescription has not been declared on
(LaTeX3)        line 6.
```

**Comment by josephwright** [comment]
I think I've mentioned before that `\GetIdInfo` is likely today a rather awkward historical thing. However, fixing this particular issue is easy enough.

**Comment by u-fischer** [comment]
@josephwright  tell it  @eg9. I got the error while checking a document loading xpatch ;-).

**Comment by josephwright** [comment]
Fixed by 3feb03dd5ec34c70db60d10de45b170a76bfc535
